cobigen-cli/cli-systemtest/src/test/java/com/devonfw/cobigen/cli/systemtest/AdaptTemplatesCommandIT.java

```diff
@@ -70,6 +58,49 @@ public void adaptTemplatesTest() throws Exception {
     // check if context configuration exists
     assertThat(templateSet.resolve(ConfigurationConstants.TEMPLATE_RESOURCE_FOLDER)).exists();
     assertThat(templateSet.resolve(ConfigurationConstants.TEMPLATE_RESOURCE_FOLDER)
-        .resolve(ConfigurationConstants.CONTEXT_CONFIG_FILENAME)).exists();
+        .resolve(ConfigurationConstants.TEMPLATE_SET_CONFIG_FILENAME)).exists();
+
+  }
+
+  /**
+   * Checks if adapt-templates command successfully created cobigen templates folder and its sub folders
+   *
+   * @throws Exception test fails
+   */
+  @Test
+  public void adaptTemplatesTest() throws Exception {
```

Comment:
I'm not sure if we still need this test, as a CobiGen update should only adapt template-sets in the future.
What do you think about this @maybeec ?